adobe_vipm/flows/global_customer.py

```diff
@@ -427,6 +441,7 @@ def create_gc_agreement_subscription(
     gc_agreement_id,
     buyer_id,
     item,
+    prices=None
```

Comment:
Didn't get this `None`. are there any cases when we shouldn't create gc subscription without proper pricing?